test/Faker/Provider/BaseTest.php

```diff
@@ -565,6 +566,20 @@ public function testRandomElements(): void
         self::assertCount(3, $allowDuplicates);
         self::assertContainsOnly('string', $allowDuplicates);
     }
+
+    /**
+     * @requires PHP 8.1
+     */
+    public function testRandomElementWithEnum(): void
+    {
+        require_once __DIR__ . '/../../Fixture/Enum/BackedEnum.php';
+
```

Comment:
```suggestion
```

You can remove the `require_once` statement here since we have autoloading configured, see `composer.json`:

https://github.com/FakerPHP/Faker/blob/1a735f050e3246fadccdf8f684122b1f97e4bddb/composer.json#L36